scripts/nolints.txt

```diff
@@ -179,18 +179,6 @@ apply_nolint locally_finite.realizer doc_blame has_inhabited_instance
 apply_nolint dfinsupp.pre doc_blame
 apply_nolint dfinsupp.zip_with_def unused_arguments
 
--- data/equiv/denumerable.lean
```

Comment:
Note that we have a script which automatically updates this file once a day, so it's not necessary to make changes to this file directly.

Reply:
Yeah I figured out. But wasn't I running the risking of malforming a docstring because of an out-of-date `nolint.txt`?

Reply:
That's true, it does serve as a check that you've gotten all the doc strings. Usually I just put `#lint` at the bottom of the file before committing and that's been good enough so far.